Content/src/SAFE.App.Client/package.default.json

```diff
@@ -0,0 +1,28 @@
+{
+    "private": true,
+    "scripts": {
+        "start": "webpack-dev-server",
+        "build": "webpack -p"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.6.4",
+        "@babel/plugin-transform-runtime": "^7.6.2",
+        "@babel/preset-env": "^7.9.6",
```

Comment:
is preset-env the only change in NPM dependencies between default and minimal? Should we review other dependencies as well?

Reply:
We should - I would suggest someone that knows more than I do about webpack and bable though!